gradio/events.py

```diff
@@ -348,6 +348,7 @@ def __init__(
         trigger_after: int | None = None,
         trigger_only_on_success: bool = False,
         doc: str = "",
+        connection: Literal["sse", "stream"] = "sse",
```

Comment:
`connection` is maybe a confusing name here, but basically we need to be able to tell which events are streaming events without relying on the event name

Reply:
Why not just a boolean `streaming` parameter?